### PR TITLE
[URGENT] Ft unit tests fixed, database migrations added

### PR DESCRIPTION
**Pull request by nodeJayS**
Bugfixes on these unit tests so they pass successfully:

- CreateComment()
- PostAsync_InvalidPost_BadRequest()
- PostAsync_ValidPost_Creates()

Initial database migration implemented

### CODE REVIEW

File: Fakebook.Posts/Fakebook.Posts.DataAccess/Repositories/PostsRepository.cs [hunk 1]
```diff
@@ -53,7 +53,8 @@ public async ValueTask<Fakebook.Posts.Domain.Models.Post> AddAsync(Fakebook.Post
         }
         public async ValueTask<Fakebook.Posts.Domain.Models.Comment> AddCommentAsync(Fakebook.Posts.Domain.Models.Comment comment)
         {
-            if (await _context.Posts.FindAsync(comment.Post.Id) is DataAccess.Models.Post post)
```
Comment (harelfishbein) [inline Fakebook.Posts/Fakebook.Posts.DataAccess/Repositories/PostsRepository.cs:56]: What is going on?
Why are you doing this?

**Review by harelfishbein — CHANGES_REQUESTED** [review]
Undo poor style choice.

**Review by nodeJayS — COMMENTED** [review]
Code coverage is poor since we included migrations file

**Review by nodeJayS — COMMENTED** [review]
LGTM

**Review by harelfishbein — APPROVED** [review]

**Review by RevatureGentry — APPROVED** [review]

**Review by KennethDavis391 — APPROVED** [review]

**Pull request merged**